Analysis/src/AnaTupleReader.cpp

```diff
@@ -20,9 +20,74 @@ std::function<Result(Args...)> bind_this(const Class& obj, Result (Class::*fn)(A
 {
     return [=](auto&& ...args) { return (obj.*fn)(std::forward<Args>(args)...); };
 }
+/*********************qui comincia ********************/
+using UncMap = std::map<std::pair<UncertaintySource, UncertaintyScale>, float>;
```

Comment:
UncMap already defined in EventTags, don't repeat it here. Same for RDF. You can do
```c++
using UncMap = EventTags::UncMap;
```